include/bout/paralleltransform.hxx

```diff
@@ -137,6 +137,20 @@ private:
   /// Cache of phase shifts for transforming from field-aligned coordinates to X-Z orthogonal coordinates
   arr3Dvec fromAlignedPhs;
 
+  /// Helper POD for parallel slice phase shifts
+  struct ParallelSlicePhase {
+    arr3Dvec phase_shift;
+    int y_offset;
+  };
+
+  /// Cache of phase shifts for the parallel slices. Slices are stored
+  /// in the following order:
+  ///     {+1, ..., +n, -1, ..., -n}
+  /// slice[i] stores offset i+1
+  /// slice[2*i + 1] stores offset -(i+1)
+  /// where i goes from 0 to (n-1), with n the number of y guard cells
+  std::vector<ParallelSlicePhase> parallel_slice_phases;
+
   arr3Dvec yupPhs; ///< Cache of phase shifts for calculating yup fields
```

Comment:
Should these two phases go as we're now using `parallel_slice_phases`? Also I know this is meant for internal use but `parallel_slice_phases` doesn't match our usual naming convention, is that ok?

Reply:
Yes, I had just left them in while I was doing the initial testing. Yes, `camelCase` is the usual here, will change